assets/js/modules/analytics/components/dashboard/LegacyAnalyticsDashboardWidgetPopularPagesTable.js

```diff
@@ -20,90 +20,90 @@
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { getTimeInSeconds, numberFormat, getModulesData } from '../../../../util';
+import Data from 'googlesitekit-data';
+import { getTimeInSeconds, numberFormat } from '../../../../util';
 import { isDataZeroForReporting, getTopPagesReportDataDefaults } from '../../util';
 import withData from '../../../../components/higherorder/withdata';
 import { TYPE_MODULES } from '../../../../components/data';
 import { getDataTableFromData, TableOverflowContainer } from '../../../../components/data-table';
 import PreviewTable from '../../../../components/preview-table';
 import Layout from '../../../../components/layout/layout';
+import { STORE_NAME } from '../../datastore/constants';
 
-class LegacyAnalyticsDashboardWidgetPopularPagesTable extends Component {
-	static renderLayout( component ) {
-		return (
-			<div className="
-				mdc-layout-grid__cell
-				mdc-layout-grid__cell--span-6-desktop
-				mdc-layout-grid__cell--span-4-tablet
-			">
-				<Layout
-					className="googlesitekit-popular-content"
-					footer
-					footerCtaLabel={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
-					footerCtaLink={ getModulesData().analytics.homepage }
-					fill
-				>
-					{ component }
-				</Layout>
-			</div>
-		);
-	}
+const { useSelect } = Data;
 
-	render() {
-		const { data } = this.props;
+const RenderLayout = ( { children } ) => {
+	const serviceURL = useSelect( ( select ) => select( STORE_NAME ).getServiceURL() );
+	return (
+		<div className="
+			mdc-layout-grid__cell
+			mdc-layout-grid__cell--span-6-desktop
+			mdc-layout-grid__cell--span-4-tablet
+		">
+			<Layout
+				className="googlesitekit-popular-content"
+				footer
+				footerCtaLabel={ _x( 'Analytics', 'Service name', 'google-site-kit' ) }
+				footerCtaLink={ serviceURL }
+				fill
+			>
+				{ children }
+			</Layout>
+		</div>
+	);
+};
 
-		if ( ! data || ! data.length ) {
-			return null;
-		}
+function LegacyAnalyticsDashboardWidgetPopularPagesTable( { data } ) {
```

Comment:
Looks like you refactored this component, but didn't actually change the URL as required per the ACs.